frontend/src/components/IssueList/IssueList.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import {css} from 'aphrodite';
+import S from './styles';
+
+export default (props) => {
+  const items = props.items.map( (item, i) => (
+    <li className={css(S.issue_style)} key={i}>
+      {item.title}
+    </li>
+  ));
+  return (
```

Comment:
I'm not sure but it might be nicer to look at if we put it all in one component and remove the need of prerendering the items. But this is what I do, feel free to ignore....

Reply:
Not sure what you mean, can you explain a bit more?